service_extensions/callouts/add_header/credentials/localhost.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I personally feel this is a security risk to include private keys in GitHub. We should document how to generate such key + certificate and how to load it, but it shouldn't be part of the committed code (some users may just reuse it which is super bad practice).

If we need the credentials for the tests to pass we could generate them on the fly or keep them in the secrets repository and pass as ENV variable.

Reply:
I am OK to leave these certificates as to unblock this PR, but we should keep in mind that a different approach should be considered in the future.

Reference: similar example already exists in appengine samples in this repo ([see here](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/appengine/standard_python3/pubsub/data/privatekey.pem))